diff --git a/node_modules/ajv/dist/ajv.bundle.js b/node_modules/ajv/dist/ajv.bundle.js
index e4d9d15..7a3e3b6 100644
--- a/node_modules/ajv/dist/ajv.bundle.js
+++ b/node_modules/ajv/dist/ajv.bundle.js
@@ -5145,7 +5145,6 @@ module.exports = function (data, opts) {
 'use strict';
 
 var traverse = module.exports = function (schema, opts, cb) {
-  // Legacy support for v0.3.1 and earlier.
   if (typeof opts == 'function') {
     cb = opts;
     opts = {};
