diff --git a/node_modules/dunder-proto/set.js b/node_modules/dunder-proto/set.js
index 6085b6e..3d2588f 100644
--- a/node_modules/dunder-proto/set.js
+++ b/node_modules/dunder-proto/set.js
@@ -24,7 +24,6 @@ module.exports = hasProtoMutator && (
 // eslint-disable-next-line no-extra-parens
 	(!!desc && typeof desc.set === 'function' && /** @type {import('./set')} */ (callBind([desc.set])))
 	|| /** @type {import('./set')} */ function setDunder(object, proto) {
-		// this is node v0.10 or older, which doesn't have Object.setPrototypeOf and has undeniable __proto__
 		if (object == null) { // eslint-disable-line eqeqeq
 			throw new $TypeError('set Object.prototype.__proto__ called on null or undefined');
 		}
