diff --git a/node_modules/eslint/lib/rules/no-mixed-requires.js b/node_modules/eslint/lib/rules/no-mixed-requires.js
index 9e7b803..2a7aa5a 100644
--- a/node_modules/eslint/lib/rules/no-mixed-requires.js
+++ b/node_modules/eslint/lib/rules/no-mixed-requires.js
@@ -75,7 +75,6 @@ module.exports = {
             /*
              * This list is generated using:
              * `require("repl")._builtinLibs.concat('repl').sort()`
-             * This particular list is as per nodejs v0.12.2 and iojs v0.7.1
              */
             return [
                 "assert", "buffer", "child_process", "cluster", "crypto",
