diff --git a/node_modules/fs.realpath/LICENSE b/node_modules/fs.realpath/LICENSE
index 5bd884c..af9cefd 100644
--- a/node_modules/fs.realpath/LICENSE
+++ b/node_modules/fs.realpath/LICENSE
@@ -17,7 +17,6 @@ IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 ----
 
 This library bundles a version of the `fs.realpath` and `fs.realpathSync`
-methods from Node.js v0.10 under the terms of the Node.js MIT license.
 
 Node's license follows, also included at the header of `old.js` which contains
 the licensed code:
